Add playbook to cleanup copr scratch repos

ehelms opened the pull request.

evgeni commented on obal/data/cleanup_copr.yml:
```diff
@@ -0,0 +1,17 @@
+  gather_facts: no
+  tasks:
+    - name: 'Get Copr repositories'
+      command: "copr-cli list"
```
should this be an explicit `copr-cli list @theforeman`?

evgeni commented on obal/data/cleanup_copr.yml:
```diff
@@ -0,0 +1,17 @@
+      register: copr_repos
+
+    - set_fact:
+        scratch_repos: "{{ copr_repos.stdout.split('\n') | select('match', '^Name:') | list | replace('Name: ', '') }}"
```
the fact should be `copr_scratch_repos`, right?

evgeni commented on obal/data/cleanup_copr.yml:
```diff
@@ -0,0 +1,17 @@
+        scratch_repos: "{{ copr_repos.stdout.split('\n') | select('match', '^Name:') | list | replace('Name: ', '') }}"
+
+    - set_fact:
+        scratch_repos: "{{ copr_scratch_repos | select('match', '.+scratch.+') | list }}"
```
the fact should be `copr_scratch_repos`, right?

evgeni commented:
do we want a `obal cleanup` action for that? or is that very copr specific and you'd use `ansible-playbook` for that?

ehelms commented:
I added an action and updated the variables. This cleanup is currently copr specific. I added 'cleanup-copr' but I wonder if `copr cleanup` should be the command?

evgeni commented:
`copr cleanup` is something `obal` would not understand today, as it expects the action to be a single world and all following to be package names. so I'd stick with the current name for now.

`ansible-lint` is unhappy, but LGTM otherwise

ehelms commented:
Updated to fix the linting errors
